vizro-core/examples/_chart-gallery/README.md

```diff
@@ -18,7 +18,7 @@ The dashboard is still in development. Below is an overview of the chart types f
 | Barcode               | ❌     | Distribution             |
```

Comment:
Reminder for myself to rename the folder after approvals. Otherwise difficult to track actual changes.

Reply:
N.B. just note it's `visual-vocabulary` with one `l`.